wca-regulations.md

```diff
@@ -76,7 +76,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 2e3) Competitors who have no citizenship may compete as "Stateless".
 - 2f) Competitors must obey venue rules and conduct themselves in a considerate manner.
 - 2g) Competitors must remain quiet when inside the designated competition area. Talking is permitted, but must be kept at a reasonable level, and away from competitors who are actively competing.
-    - 2g3) Competitors in a Competitor Waiting Area must not communicate with each other about the scrambled states of the puzzles of the round in progress. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate.
+    - 2g3) Competitors must not communicate with each other about the scrambled states of the puzzles of the group/round in progress. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attmpet may discuss their solution with other competitors that have finished their attempt (see the regulations for [Article E](regulations:article:E) and the guidelines for [Article E](guidelines:article:E)).
```

Comment:
```suggestion
    - 2g3) Competitors must not communicate with each other about the scrambled states of the puzzles of the group/round in progress. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attmpet may discuss their solution with others (see [Article E](regulations:article:E)).
```

I feel like talking to just guests (e.g. parents) should be allowed too. Also, we do not usually link to guidelines.

Reply:
```suggestion
    - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group's puzzles unless both competitors are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves and 3x3x3 Multi-Blind competitors may communicate with each other about the scramble sequence(s) for an individual attempt provided both have finished that attempt (see [Article E](regulations:article:E) and [Article H](regulations:article:H)).
```

While the previous use of "competitor waiting area" was problematic as it did not cover the possibility of competitors communicating at solving stations (especially while fixed seating is in use), its removal make the regulation too broad. "Competitor" alone could be interpreted as being any competitor at the competition, precluding delegates, judges etc. from communicating necessary information, for example while investigating a duplicate scramble. I have attempted a draft that keeps the restricted group as narrowly defined as possible. 

"Group" is preferable to "round" as it is more specific and limiting. "Group/round" is unnecessary as per 1h competitors in a round that is not subdivided can still be considered members of a single group.

"Not eligible for any further attempts in the group" is a bit awkward, but simply saying "finished their attempts" doesn't capture scenarios in which there are DNSs. I've not tried to capture this drafting nuance in the exception.

As multiblind also has individually scheduled attempts I have added this as a further exception. 

Should we consider a broader sanction here (disqualification from further events), as the existing one may be ineffective in certain cases? For instance, 2j2 means a competitor telling another about scramble 5 of a final round after performing that scramble receives only a very limited punishment from the application of this regulation. The broader power to disqualify further under 2j without precondition, or 2k5 for breaking this specific regulation still exists if necessary.

Reply:
```suggestion
    - 2g3) Competitors must not communicate with each other about the scrambled states of the puzzles of the group/round in progress. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share pictures of the scramble with anyone who is not actively competing in the relevant attempt.
```

Reply:
Antonio's is in line with the following idea:

The initial idea was to clarify that competitors shouldn't be punished for sharing pictures of their solution while the attempt is still ongoing (as other competitors shouldn't have access to their phones anyway). It's not a good idea to allow competitors to talk about their solutions, as it is not easy to regulate and control.

@Nevseros @ohexter What do you think about this?

Reply:
Apologies for the slow response - I think keeping the wording of the exception focused on scramble sequences rather than solutions is good as that's what the regulation is about. 

I stand by the points in my April 3 comment, which it's a little disappointing to see were not resolved or addressed by the version that has now been committed and approved.